packages/contracts/util/exchange_wrapper.ts

```diff
@@ -158,6 +158,20 @@ export class ExchangeWrapper {
         _.each(tx.logs, log => wrapLogBigNumbers(log));
         return tx;
     }
+    public async cancelOrdersBeforeAsync(
+    timestamp: BigNumber,
+    from: string,
+    ): Promise<TransactionReceiptWithDecodedLogs> {
+        const params = formatters.createCancelOrdersBefore(timestamp);
```

Comment:
I think we can remove this formatter and just use timestamp directly, since it's only one argument.